src/web/src/components/AboutFooter.test.tsx

```diff
@@ -1,11 +1,11 @@
 import { render, cleanup } from '@testing-library/react';
-import AboutFooter from './AboutFooter';
 
 afterEach(cleanup);
 
 jest.mock('./GitHubContributorCard', () => () => <div>GitHubContributorCard</div>);
 
 it('renders correctly', () => {
+  const AboutFooter = require('./AboutFooter').default;
```

Comment:
for some reason, the `GitHubContributorCard` mock is not hoisted before the import of `AboutFooter` and returns null. I had to use `require` to make it work.

Reply:
This needs a comment above it explaining why, so that we don't remove it.